app/views/business_mailer/new_developer_profile.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>New developers at Railsdevs.com!</h1>
+
+<p><%= @business.name %>,</p>
+<p>Here's your digest of new developers available for hire at Railsdevs.com:</p>
+<hr />
+<% @developers.each do |dev| %>
+  <p><%= link_to dev.name, dev %>: <%= dev.hero %></p>
+  <hr />
+<% end %>
```

Comment:
The horizontal rules are a little aggressive without any CSS. Let's change them to a `<ul>` and `<li>` instead.